Forta-Bot-Challenge-4/COMP01/src/thresholdCache/thresholdCache.ts

```diff
@@ -0,0 +1,108 @@
+import { LRUCache } from "lru-cache";
+import { ethers } from "ethers";
+
+import { BigNumber } from "@ethersproject/bignumber";
+
+export let THRESHOLD: any = ethers.BigNumber.from("10");
+
+export const amountCache: LRUCache<string, { timestamp: number; amount: any }> = new LRUCache({
```

Comment:
```suggestion
export const amountCache: LRUCache<string, { timestamp: number; amount: string }> = new LRUCache({
```
Currently, the type of `amount` should be `string`. However, could it not be type `BigNumber` to save a bit of conversion you're having to do?